Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/skylanders/SkylanderConfig.java

```diff
@@ -0,0 +1,30 @@
+package org.dolphinemu.dolphinemu.features.skylanders;
+
+import android.util.Pair;
```

Comment:
This is now unused as far as I can tell.

Reply:
Still used for Load/Create, where the pair is <Integer,String>,  integer is the slot on the portal and string is the name